### DISCUSSION TITLE
(minor) Start streaming when starting the SP or selecting the source, or when displaying the Remote window

### DISCUSSION THREAD

**Discussion by bradut** [GerryDazoo/SlingerPlayer#16] [Ideas]
I think that it might be helpful to be able to start streaming when the SP starts (similar to SP 1.5), and also when a another Slingbox is selected.
If this is not possible, then, if not started already, the stream may start automatically when the Remote button is clicked.

(As of now, from the Remote window, only the Restart button can control the server when the stream is not started).

**Comment by GerryDazoo** [comment]
You can start a stream when the player starts by putting the URL on the command line.
i.e. SlingerPLayer.exe http://ip:port/urlbase/sbName

I'll look into starting the stream when the remote button is pressed. Seems reasonable.
